### PR TITLE
Bump version to 1.1.0

### PR DESCRIPTION
**Pull request by JingweiZhang12**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Note that this version depends on mmdet v3.3.0rc6 since using the refactored DETR series in BEVFusion.

## BC-breaking (Optional)

Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/notes/changelog.md [hunk 1]
```diff
@@ -1,5 +1,54 @@
...
+- Support [Cylinder3D](https://arxiv.org/pdf/2011.10033.pdf) (#2291, #2344, #2350)
+- Support [MinkUnet](https://arxiv.org/abs/1904.08755) (#2294, #2358)
+- Support [SPVCNN](https://arxiv.org/abs/2007.16100) (#2320，#2372)
+- Support [TR3D](https://arxiv.org/abs/2302.02858) detector in `projects` (#2274)
+- Support the inference of [BEVFusion](https://arxiv.org/abs/2205.13542) in `projects` (#2175)
+- Support [DETR3D](https://arxiv.org/abs/2110.06922) in `projects` (#2173)
+- Support PolarMix and LaserMix augmentation (#2265, #2302)
+- Support loading annotation of panoptic segmentation (#2223)
+- Support panoptic segmentation metric (#2230)
+- Add inferencer for LiDAR-based, monocular 3d and multi-modality detection (#2208, #2190, #2342)
```
Comment (ZCMax) [inline docs/en/notes/changelog.md:25]: Add inferencer for LiDAR-based, monocular and multi-modality 3D detection

File: docs/en/notes/changelog.md [hunk 2]
```diff
@@ -1,5 +1,54 @@
...
+- Add inferencer for LiDAR-based segmentation (#2304)
+
+#### Improvements
+
+- Support `lazy_init` for CBGSDataset (#2271)
+- Support generating annotatin file for test set on Waymo  (#2180)
+- Enhance the support for SemanticKitti (#2253, #2323)
+- File I/O migration and reconstruction (#2319)
+- Support `format_only` option for Lyft, NuScenes and Waymo (#2333, #2151)
```
Comment (ZCMax) [inline docs/en/notes/changelog.md:34]: for Lyft, NuScenes and Waymo dataset

File: docs/en/notes/changelog.md [hunk 3]
```diff
@@ -1,5 +1,54 @@
...
+- Replace np.transpose by torch.permute to speed up (#2277)
```
Comment (ZCMax) [inline docs/en/notes/changelog.md:35]: Replace `np.transpose` by `torch.permute` to speed up

File: docs/en/notes/changelog.md [hunk 4]
```diff
@@ -1,5 +1,54 @@
...
+
+#### Bug Fixes
+
+- Fix the reversal of length and width when drawing heatmap in CenterFormer (#2362)
+- Deprecate old type alias due to new version of numpy (#2339)
+- Lose trimesh version requirements to fix numpy random state (#2340)
```
Comment (ZCMax) [inline docs/en/notes/changelog.md:41]: Remove `trimesh` version limitations to fix numpy random state

File: docs/en/notes/changelog.md [hunk 5]
```diff
@@ -1,5 +1,54 @@
...
+- Fix the device mismatch error in CenterPoint (#2308)
+- fix bug of visuzatioin when there are no bboxes (#2231)
```
Comment (ZCMax) [inline docs/en/notes/changelog.md:43]: fix -> Fix

File: docs/en/notes/changelog_v1.0.x.md [hunk 6]
```diff
@@ -1,5 +1,23 @@
 # Changelog of v1.0.x
 
+### v1.0.0rc8 (26/3/2023)
+
+#### Improvements
+
+- Add BN in FPN to avoid loss Nan in MVXNet (#2282)
+- Update s3dis_data_utils.py (#2232)
```
Comment (ZCMax) [inline docs/en/notes/changelog_v1.0.x.md:8]: Update `s3dis_data_utils.py`

**Review by ZCMax — APPROVED** [review]

File: docs/en/notes/changelog.md [hunk 7]
```diff
@@ -1,5 +1,55 @@
...
+- Lose `trimesh` version requirements to fix numpy random state (#2340)
+- Fix the device mismatch error in CenterPoint (#2308)
+- Fix bug of visualization when there are no bboxes (#2231)
+- Fix bug of counting ignore index in IOU in segmentation evaluation (#2229)
+
+#### Contributors
+
+A total of 14 developers contributed to this release.
+
+@ZLTJohn, @SekiroRong, @shufanwu, @vansin, @@triple-Mu, @404Vector, @filaPro, @sunjiahao1999, @Ginray, @Xiangxu-0103, @JingweiZhang12, @DezeZhao, @ZCMax, @roger-lcc
```
Comment (sunjiahao1999) [inline docs/en/notes/changelog.md:51]: Should be @triple-Mu?

**Review by sunjiahao1999 — APPROVED** [review]

**Pull request merged**